mbed_lstools/lstools_base.py

```diff
@@ -110,7 +110,7 @@ def list_mbeds_ext(self):
     def list_mbeds(
             self, fs_interaction=FSInteraction.BeforeFilter,
             filter_function=None, unique_names=False,
-            read_details_txt=False):
+            read_details_txt=None):
```

Comment:
Could you revert this change? It seems like a "style-like" thing to me. I don't think this code explicitly looks for `None` instead of any falsey value

Reply:
Oops good catch. That was from when I had initially renamed the parameter.